### ISSUE TITLE
citations ending in 'i', continue part 1

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/MWinflect#18]
This is analogous to #17.

Here, we extend the identification of stems ending in 'i' to those lexnorm-all2 cases where 
* the citation ends in 'i'
* the lexnorm is exactly one of
  * f:n   3 cases
  * m:n   11 cases
  * m:f  122
  * m:f:n 1689

This handles about  1800 additional records of lexnorm-all2 

After removing duplications, there are additional stems added:
* 1559 additional stems with m_i model
* 1820 additional stems with f_i model
* 1663 additional stems with n_i model.

for a total of 5042 additional stem-model cases ending in 'i'.

There remain 158 records of lexnorm-all2 with stems ending in 'i' which will be accounted for later.
Some examples:
```
25903	Ayuzpati	Ayuz-pati	m:f#tnI:n
28537	i	i	LEXID=pron,STEM=idam
232753	saptati	saptati	LEXID=card,STEM=saptati
```

The declension of these additional 5042 cases requires no modification to the declension algorithms.

**Comment by funderburkjim** [comment]
I had forgotten, or not known of, the SanskritSubanta.     Thanks for mentioning.

Have worked with @drdhaval2785 in comparing
his SanskritVerb verb forms to those generated by my code. This work done back in 2016. See https://github.com/funderburkjim/elispsanskrit/issues.   Will no doubt revisit this work in this MWinflect project when I get to verb conjugations.

**Comment by drdhaval2785** [comment]
I will also need to revisit SanskritSubanta. It was merged with SanskritVerb at one point of time.

**Comment by gasyoun** [comment]
> also need to revisit 

Yes, indeed. MSS will not run away if you take some rest with us, will they?
